### PR TITLE
Pwned - Bug fix in Reason field in the context

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17673

## Does it break backward compatibility?
   - Yes
current context output had not only wrong data, but the data was going to a different path, which was not mentioned. so I edited the exisiting always empty context field to the correct one.
line 274 in yml file

## Must have
- [x] Tests
- [x] Documentation - https://github.com/demisto/etc/issues/17764
- [x] Code Review

### CODE REVIEW

File: Integrations/integration-Pwned.yml [hunk 1]
```diff
@@ -182,7 +189,7 @@ script:
     - contextPath: Account.Email.Compromised.Vendor
       description: For comporomised emails, the vendor that made the decision
       type: string
-    - contextPath: Account.Email.Reason
+    - contextPath: Account.Email.Compromised.Reason
```
Comment (yuvalbenshalom) [inline Integrations/integration-Pwned.yml]: since we are changing the path anyway, maybe we can add a more meaningful name than reason?
or is it the agreed convention for us to say "who found this?"
@kirbles19 lets check this.

Comment (yuvalbenshalom) [reply Integrations/integration-Pwned.yml]: @Arsenikr proposed to use "reporters"

**Comment by bakatzir** [comment]
@yuvalbenshalom this is break of BC, so if approved will need to force merge

File: Integrations/integration-Pwned.yml (hunk 1 above)
Comment (kirbles19) [reply Integrations/integration-Pwned.yml]: Still relevant?

File: Integrations/integration-Pwned.yml [hunk 2]
```diff
@@ -19,25 +19,24 @@ configuration:
...
 
-    var sendRequest = function(url) {
+    function sendRequest(url) {
         var res = http(
             url,
             {
                 Method: 'GET',
                 Headers: {
                     'Accept': ['application/json'],
-                    'User-Agent' : ['DBOT-API'],
```
Comment (yuvalbenshalom) [inline Integrations/integration-Pwned.yml]: why did you remove?

File: Integrations/integration-Pwned.yml (hunk 1 above)
Comment (bakatzir) [reply Integrations/integration-Pwned.yml]: @kirbles19  no ty, we're good

File: Integrations/integration-Pwned.yml (hunk 2 above)
Comment (bakatzir) [reply Integrations/integration-Pwned.yml]: by mistake. reverting.

**Comment by bakatzir** [comment]
@kirbles19 see .yml:
https://github.com/demisto/content/edit/pwned_domains_fix/Integrations/integration-Pwned.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F3789

**Comment by kirbles19** [comment]
@bakatzir done reviewing .yml
